src/Arcus.Observability.Telemetry.Serilog/VersionEnricher.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Reflection;
+using Serilog.Core;
+using Serilog.Events;
+
+namespace Arcus.Observability.Telemetry.Serilog
+{
+    /// <summary>
+    /// Enrichment on log events with the current runtime version (i.e. 'version' = '1.0.0').
+    /// </summary>
+    public class VersionEnricher : ILogEventEnricher
+    {
+        private readonly string _assemblyVersion;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="VersionEnricher"/> class.
+        /// </summary>
+        public VersionEnricher()
+        {
+            var executingAssembly = Assembly.GetExecutingAssembly();
+            if (executingAssembly == null)
+            {
+                throw new InvalidOperationException(
+                    "Cannot enrich the log events with a 'Version' because the version of the current executing runtime couldn't be determined");
+            }
+            
+            _assemblyVersion = 
+                executingAssembly.GetCustomAttribute<AssemblyVersionAttribute>()?.Version
```

Comment:
Thinking out loud, shouldn't we use InformationalVersion as most interesting one?

We should try to get this version:
![image](https://user-images.githubusercontent.com/4345663/74712800-7d714980-5227-11ea-89be-48d0aeffa72a.png)

Reply:
If I'm correct, the `AssemblyVersionAttribute` will reflect the `Product version`. So that's already OK.

Reply:
Can we test this in some way? Preferrably but alternatively manually.

Reply:
Ok, tested it with a EventGrid .dll:
![image](https://user-images.githubusercontent.com/9039753/74717156-2cb21e80-5230-11ea-92bc-061f6ce97d9a.png)

Apperently, the `AssemblyInformationalVersionAttribute` is indeed the right one.

Reply:
Ok, good that we've tested it to make sure it's correct!